xclip.c

```diff
@@ -465,7 +465,10 @@ doOut(Window win)
 		else {
 		    /* no fallback available, exit with failure */
 		    char *atom_name = XGetAtomName(dpy, target);
-		    fprintf(stderr, "Error: target %s not available\n", atom_name);
+		    fprintf(stderr, "xclip::Error: Cannot performance ConvertSelection to target %s\n", atom_name);
```

Comment:
-1. First of all, if xclip should include "xclip" in error messages, this change should be done in the entire project. For example, error messages such "Error: Could not allocate memory" should also be updated. Also, I do not like the double colon format. Better to use something like "xclip: Error: ".

Reply:
Should probably be a separate PR if we should do this.

Reply:
Agree.
Moved the related code in xcprint.c. All printing-error code are put in one file and added with a prefix "xclip: Error: ".